### DISCUSSION TITLE
profile privacy settings

### DISCUSSION THREAD

**Discussion by tamfelix** [community/community#123156] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

would it be posssible have a semi-private option mode,
where other users can see your start page  (pins, followers, achievements, contributions ) APART from
-contribution activity timeline of every action taken

**Comment by iTechTop** [comment]
🤔GitHub currently offers two modes for profiles: private and public. There isn't a semiprivate option that reveals selected sections while concealing others, such as contribution activity timeline. In cases where partial exposure proves necessary, creative approaches combining multiple avenues achieve comparable outcomes.

One feasible strategy entails crafting a secondary profile acting as a landing page featuring sanctioned sections alongside limited interaction scopes. Subsequently, employ organizational accounts or hidden repositories to isolate restricted elements, thereby balancing visibility and discretion.

#### Landing Page Profile

Establish a separate account solely serving as a landing page. Populate authorized portions, e.g., pinned repositories, followed users, badges, and biography. Grant unrestricted access to this profile, encouraging visitors to traverse permitted segments openly.

#### Private Organization Account

Set up an organizational account operating behind closed doors, exclusively housing clandestine projects. Invite trusted colleagues and collaborators, regulating participation levels judiciously. Organizations accommodate restricting membership visibilities besides confining repository access.

#### Hidden Repositories

Optimize repositories for selective viewing by toggling their visibilities appropriately. Two viable modes exist: public and private. Enlist secret repositories for confidential undertakings, granting entry sparingly. Meanwhile, preserve less delicate ventures publicly, retaining transparent contribution histories devoid of granular intricacies.

Regrettably, achieving precise alignment matching your preference isn't attainable within GitHub's prevailing framework. Applying inventive tactics delivers satisfactory compromises aligning closely with anticipated behaviors.

**Comment by caturndev** [comment]
I need this but reverse, where people can see everything BUT your following and followers.
